wIndex/Diagnostics-FindAllFormulas.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Diagnostics - Find All Workbook Formulas
+filename: "Diagnostics-FindAllFormulas.md"
+layout: custom
+keywords: []
+headings: ["Overview", "Text Formula", "Text All", "SQL Query", "XML"]
+links: []
+image_dir: "DiagnosticsFindAllFormulas"
+images: [
+    {file: "FindAllFormulas", type: "png", site: "Addin", cat: "Diagnostics", sub: "FindAllWorkbookFormulas", report: "", ribbon: "", config: ""}, 
+    {file: "TextFormula", type: "png", site: "Addin", cat: "Diagnostics", sub: "FindAllWorkbookFormulas", report: "", ribbon: "", config: ""}, 
+    {file: "TextAll", type: "png", site: "Addin", cat: "Diagnostics", sub: "FindAllWorkbookFormulas", report: "", ribbon: "", config: ""}, 
+    {file: "SQLQuery", type: "png", site: "Addin", cat: "Diagnostics", sub: "FindAllWorkbookFormulas", report: "", ribbon: "", config: ""}, 
+    {file: "SQLOutput", type: "png", site: "SSMS", cat: "Code", sub: "", report: "", ribbon: "", config: ""}, 
+    {file: "XML", type: "png", site: "Addin", cat: "Diagnostics", sub: "FindAllWorkbookFormulas", report: "", ribbon: "", config: ""}
+    ]
+]
+description: The Diagnostic command Find All Workbook Formulas will display information about every Interject formula in the active workbook. Depending on which line is first in the input field, it will output to several different formats.
+
+---
+* * *
+
+## Overview
+
+The Diagnostic command "Find All Workbook Formulas" will display information about every Interject formula in the active workbook. Depending on which line is first in the input field, it will output to several different formats.
+
+![](/images/DiagnosticsFindAllFormulas/FindAllFormulas.png)
+<br>
+
+
+### Text Formula
+
+Clicking on **Execute Selected Action** when "text_formula" is first in the input field, and the **Results** will display information about all the Interject formulas used in the workbook:
```

Comment:
I think the 'and' should be left out of this sentence, either that or change 'Clicking' to 'Click'